cloudclient/middleware.go

```diff
@@ -58,7 +58,7 @@ func handleHTTPResponseToGRPCRequest(errInput error) error {
 		)
 	}
 	// Other HTTP responses to gRPC requests are assumed to be transient.
-	return status.Errorf(codes.Unavailable, errorMessage)
+	return status.Errorf(codes.Unavailable, "%s", errorMessage)
```

Comment:
If `--fix` isn't suggesting this, I will 😉 
```suggestion
	return status.Error(codes.Unavailable, errorMessage)
```

Reply:
nice